### ISSUE TITLE
Default derivation produces non-working instance.

### ISSUE THREAD

**Issue by ShaneEverittM** [wackywendell/primes#7]
If you create a `Sieve` like so:
```rust
use primes::{PrimeSet, Sieve};
let mut s = Sieve::default();
dbg!(s.get(5));
```
The program will never terminate. Switching to `Sieve::new` fixes it.

**Comment by EMBradley** [comment]
I’m on mobile right now so I can’t submit a fix until later, but I think I know what’s wrong here.

The library derives `Default` for `Wheel30` and `Sieve`, but the derived defaults aren’t useful. The [derived implementation of `Default`](https://doc.rust-lang.org/std/default/trait.Default.html#Derivable) will take the default values for each field in `Wheel30` and `Sieve`, resulting in a wheel with base 0 and index 0 (not what you get from `Wheel30::new()`) and a sieve with an empty list of known primes, an empty heap of known composites, and a default wheel (also not what you get from `Sieve::new()`).

Solution: explicitly implement `Default` with the functions `Wheel30::default()` and `Sieve::default()` simply calling their respective `new()` functions, rather than deriving `Default`.

**Comment by wackywendell** [comment]
Ah! Good catch. I implemented your suggestion in 0d0298f377cea28ce76a5f735b4d2f0da0ff9c58, so should be all set!

I'll close this now, but feel free to check it out and reopen if I missed something.
